os.path/os/path.py

```diff
@@ -79,6 +79,13 @@ def islink(path):
     except OSError:
         return False
 
+def realpath(path):
+    import ffilib
+    libc = ffilib.libc()
+    if isinstance(path, str) or isinstance(path, bytes):
+        realpath_ = libc.func("s", "realpath", "ss")
+        return realpath_(path, 0)
+    raise TypeError("expected str or bytes, not {}".format(type(path)))
```

Comment:
Please just use `raise TypeError`, no bytes for long error messages ;-).

Reply:
I see :)